CONTRIBUTING.md

```diff
@@ -1,29 +1,29 @@
 This guide describes how to develop and contribute pull requests to this connector. The focus is currently on how to
 develop and test the connector, either via a local install of Confluent Platform or of the regular Kafka distribution.
 
-Before beginning, you will need to install Java (either version 8, 11, or 17) and also have a MarkLogic instance 
-available. It is recommended to use 11 or 17, as Confluent has deprecated Java 8 support in Confluent 7.x and is 
-removing it in Confluent 8.x. See [the Confluent compatibility matrix](https://docs.confluent.io/platform/current/installation/versions-interoperability.html#java)
-for more information. After installing your desired version of Java, ensure that the `JAVA_HOME` environment variable
-points to your Java installation.
+Before beginning, you will need to install Java (either version 8, 11, or 17). It is recommended to use 17, as Confluent
+has deprecated Java 8 support in Confluent 7.x and is removing it in Confluent 8.x.
+See [the Confluent compatibility matrix](https://docs.confluent.io/platform/current/installation/versions-interoperability.html#java) for more information. Additionally running Sonar requires Java 17. After
```

Comment:
Check our spark-etl config and use the pinned version of Sonar so we only need Java 11.